### DISCUSSION TITLE
Groupall & preserveSemverRanges : dependencies upgrade "missed "

### DISCUSSION THREAD

**Discussion by xuoja-acc** [renovatebot/renovate#11642] [Request Help]
Hi,

We'd like to use renovate to upgrade our depencies without pinning(preserveSemverRanges) version and using the "groupall" but :
- We'v got 4 dependencies upgrade with : `config:base , github>whitesource/merge-confidence:beta , group:all , :preserveSemverRanges`
- After pinning dependencies we'v got >20 dependencies with : `config:base , github>whitesource/merge-confidence:beta , group:all`

We'd like to have the same results with preserveSemverRanges than "pinned versions" , is it possible?

**Renovate version** : 26.21.5

**Reproduction environment** :
- Sample "project" : https://gitlab.com/ajoux.accor/renovate
- Configuration : https://gitlab.com/ajoux.accor/renovate/-/blob/debug/groupall/config-presets/groupall.json
- Sample packages.json : https://gitlab.com/ajoux.accor/renovate/-/blob/debug/groupall/package.json
- The MR without pinning : https://gitlab.com/ajoux.accor/renovate/-/merge_requests/12
- The MR after pinning the versions : https://gitlab.com/ajoux.accor/renovate/-/merge_requests/11

**Comment by viceice** [comment]
checkout https://docs.renovatebot.com/configuration-options/#rangestrategy

I think you need `bump` or `update-lockfile`

**Reply by xuoja-acc** [reply]
Thank's !

Setting the rangeStrategy to bump worked in the config.json, but didn't worked using environment variables  : https://gitlab.com/ajoux.accor/renovate/-/commit/c506d7f1a5235a062a5b3bab9cce394f4c3d2956

:confused:
